SEBrowser/Scripts/TSX/Components/EventSearch/EventSearchPreview/EventSearchPreviewPane.tsx

```diff
@@ -144,8 +145,6 @@ export default function EventPreviewPane(props: IProps) {
 
                         else if (widget.Name === 'EventSearchFileInfo')
                             return <EventSearchFileInfo key={index} eventID={props.EventID} />;
-                        else if (widget.Name === 'EventSearchHistory')
-                            return <EventSearchHistory key={index} eventID={props.EventID} />;

```

Comment:
If we're deleting this widget here, the file should probably be removed as well.

Reply:
Done